src/lib/djerba/plugins/pwgs/case_overview/plugin.py

```diff
@@ -77,3 +78,7 @@ def specify_params(self):
         self.set_ini_default(core_constants.ATTRIBUTES, 'clinical')
         self.set_priority_defaults(self.PRIORITY)
 
+    def write_results_to_json(self, data):
```

Comment:
As noted above, this method is not needed

Reply:
Thanks ! I will make the necessary changes